packages/utils/src/lib/verbose-utils.unit.test.ts

```diff
@@ -1,40 +1,41 @@
-import { describe, expect, it } from 'vitest';
+import { beforeEach, describe, expect, it, vi } from 'vitest';
+import { ui } from './logging';
 import { verboseUtils } from './verbose-utils';
 
 describe('verbose-utils', () => {
+  beforeEach(() => {
+    ui().logger.flushLogs();
+  });
   it('exec should be off by default', () => {
     const spy = vi.fn();
     verboseUtils().exec(spy);
     expect(spy).not.toHaveBeenCalled();
-    expect(console.info).not.toHaveBeenCalled();
   });
 
   it('exec should work no-verbose', () => {
     const spy = vi.fn();
     verboseUtils(false).exec(spy);
     expect(spy).not.toHaveBeenCalled();
-    expect(console.info).not.toHaveBeenCalled();
   });
 
   it('exec should work verbose', () => {
     const spy = vi.fn();
     verboseUtils(true).exec(spy);
     expect(spy).toHaveBeenCalled();
-    expect(console.info).not.toHaveBeenCalled();
   });
 
   it('logs should be off by default', () => {
     verboseUtils(false).log('42');
-    expect(console.info).not.toHaveBeenCalled();
+    expect(ui().logger.getRenderer().getLogs()).toHaveLength(0);
   });
 
   it('log should work no-verbose', () => {
```

Comment:
Quite a cryptic name. How should they work in `no-verbose` vs `verbose`?
```suggestion
  it('should not print any logs when verbose is off', () => {
```